### DISCUSSION TITLE
How to disable focus

### DISCUSSION THREAD

**Discussion by dagomez137** [zellij-org/zellij#3562] [Q&A]
When I connect and attach to an active Zellij session, I see a focus indicator with the text "My Focus and: []" at the top bar of the active pane. Additionally, the pane size may remain small from a previous session where the window size was smaller (because I keep attaching to the same session, either on a big monitor or a laptop screen).

How can I disable the focus indicator and resize the window to its maximum size?

Thanks!

**Comment by Lockszmith-GH** [comment]
use the session manager to disconnect all other sessions.

**Reply by dagomez137** [reply]
Thanks! `Ctrl-o+w` to open the session manager and then, `Ctrl+x` to "Disconnect others". 

Since I'm the only user for the session, it makes sense to attach and force disconnect others. Alternatively, creating a shortcut would be helpful. However, I don't see an option to do this in the command-line options or keybinding actions [1] as described in the manual. Is that correct?

[1] https://zellij.dev/documentation/keybindings-possible-actions
